xchange-bankera/src/main/java/info/bitrich/xchangestream/bankera/BankeraStreamingMarketDataService.java

```diff
@@ -0,0 +1,84 @@
+package info.bitrich.xchangestream.bankera;
+
+import info.bitrich.xchangestream.core.StreamingMarketDataService;
+import io.reactivex.Observable;
+import org.knowm.xchange.bankera.BankeraAdapters;
+import org.knowm.xchange.bankera.dto.BankeraException;
+import org.knowm.xchange.bankera.dto.marketdata.*;
+import org.knowm.xchange.bankera.service.BankeraMarketDataService;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trade;
+import org.knowm.xchange.exceptions.NotAvailableFromExchangeException;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+
+
+public class BankeraStreamingMarketDataService implements StreamingMarketDataService {
+
+  private final BankeraStreamingService service;
+  private final BankeraMarketDataService marketDataService;
+
+  public BankeraStreamingMarketDataService(BankeraStreamingService service, BankeraMarketDataService marketDataService) {
+    this.service = service;
+    this.marketDataService = marketDataService;
+  }
+
+  @Override
+  public Observable<OrderBook> getOrderBook(CurrencyPair currencyPair, Object... args) {
+    BankeraMarket market = getMarketInfo(currencyPair);
```

Comment:
Does Bankera send the full order book with every update? No deltas?

Reply:
Yes it does send full orderbook for selected market.